### PR TITLE
refactor: Use template literal instead of '+' operator

### PR DESCRIPTION
**Pull request by Dhoni77**
## What does this PR do?

Refactor of code:

Replace concatenation using '+' operator to template literals to make code more readable.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by zomars — COMMENTED** [review]
If we're going to take this route. Would it make sense to enable https://eslint.org/docs/latest/rules/prefer-template on this PR to prevent future disparities?

**Comment by Dhoni77** [comment]
> If we're going to take this route. Would it make sense to enable https://eslint.org/docs/latest/rules/prefer-template on this PR to prevent future disparities?

Hi @zomars  thanks for the suggestion I have added the eslint rule to prefer template literals.

**Review by Abdel-Monaam-Aouini — APPROVED** [review]

**Comment by PeerRich** [comment]
this PR will take us a while to review due to its crazy size. it touches everything

**Comment by Dhoni77** [comment]
> this PR will take us a while to review due to its crazy size. it touches everything

Sure @PeerRich Take your time👍

File: packages/config/eslint-preset.js [hunk 1]
```diff
@@ -19,29 +19,30 @@ module.exports = {
...
+    //     destructuredArrayIgnorePattern: "^_",
+    //   },
+    // ],
+    // "unused-imports/no-unused-imports": "error",
+    // "no-restricted-imports": [
+    //   "error",
+    //   {
+    //     patterns: ["lodash"],
+    //   },
+    // ],
```
Comment (zomars) [inline packages/config/eslint-preset.js:44]: Why all these are commented out? Let's revert this please.

**Review by zomars — CHANGES_REQUESTED** [review]
image

Almost done!

Left a comment to revert in there. Let's also fix the new conflicts. And lastly making sure the checks are passing. 🙏🏽

File: packages/config/eslint-preset.js (hunk 1 above)
Comment (Dhoni77) [reply packages/config/eslint-preset.js]: Uncommented! Thanks for pointing out @zomars

**Review by zomars — APPROVED** [review]
Thank you for your contribution 🙏

Looks good to merge now

Ship it

File: packages/trpc/server/middlewares/sessionMiddleware.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
   return {
     ...user,
     avatar:
-      `${WEBAPP_URL}/${user.username}/avatar.png?` + user.organizationId && `orgId=${user.organizationId}`,
+      `${WEBAPP_URL}/${user.username}/avatar.png?${user.organizationId}` && `orgId=${user.organizationId}`,
```
Comment (emrysal) [inline packages/trpc/server/middlewares/sessionMiddleware.ts:105]: This regressed :(

Comment (Dhoni77) [reply packages/trpc/server/middlewares/sessionMiddleware.ts:105]: Hi @emrysal should I create a seperate pr for this.

**Pull request merged**